ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -21,15 +21,26 @@ import com.daml.ledger.test.model.Iou.IouTransfer._
 import com.daml.ledger.test.model.IouTrade.IouTrade
 import com.daml.ledger.test.model.IouTrade.IouTrade._
 import com.daml.ledger.test.model.Test._
+import com.daml.test.evidence.tag.Security.SecurityTest
+import com.daml.test.evidence.tag.Security.SecurityTest.Property.Privacy
 
 import scala.collection.immutable.Seq
 import scala.collection.mutable
 
 class TransactionServiceVisibilityIT extends LedgerTestSuite {
+
+  val securityAsset: String => SecurityTest = (asset: String) =>
```

Comment:
Could add a helper like `securityTags` that returns a List of tags.

This could allow more compact syntax like:
```
tags = securityTag("Transaction "Transaction trees are served according to the blinding/projection rules")
```

Reply:
or changing `LedgerTestSuite` and overload this method with single tag as one of the options. Let me try

Reply:
in fact I cannot have same overloaded method with default values, so I tried to implement your suggestion, wdyt now?